Content.Shared/Fax/SharedFax.cs

```diff
@@ -34,6 +34,20 @@ public sealed class FaxUiState : BoundUserInterfaceState
     }
 }
 
+[Serializable, NetSerializable]
+public sealed class FaxFileMessage : BoundUserInterfaceMessage
+{
+    public string Content;
+    public string Name;
+    public bool OfficePaper;
+
+    public FaxFileMessage(string content, string name, bool officePaper)
```

Comment:
The name is constant so client doesn't need to send it which will save bandwidth and avoiding having to validate.